data/korath/korath missions.txt

```diff
@@ -576,3 +576,620 @@ mission "Efreti: Distress Call 8"
+			choice
+				`	(Nod.)`
+				`	(Make a motion like a ship landing and point to the Korath's mouth.)`
+			action
+				set "First Contact: Korath Exiles: agreed to bring food"
+			`	"Lasel," says the Korath, stretching out both arms toward you, fingers curled back towards its body. You make the same gesture, and they reverse their hands, linking their scaly fingers with yours. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters and take off, circling above you.`
+				decline
+
+			label alone
+			`	On the map, you draw a line between Korath space and human space. "Humans will stay here," you say, "if the Korath will stay here. Humans here, Korath here." You point to each location for emphasis.`
+				goto parting
+			label fight
+			`	They push their finger deep into the ground at a point on the map not too far from Earth. "Ka puk soru, nik ka korath."`
+			label parting
+			`	"Kai," says the Korath, and they stretch out both arms toward you, palms out. You make the same gesture and touch their hands, palm to palm. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters, take off, and hover nearby.`
+				decline
+
+
+
+mission "Korath [F01]: A Fishy Deal"
+	landing
+	name "A fishy proposition"
+	description "Transport <cargo> to <planet> by <date>. After a return trip, you'll be paid 140,000 credits. Don't get caught."
+	cargo "live fish" 15
+	# The mission is illegal because of the broader trade agreement. The player is supposed to be left with the impression that the Korath are sending back more dangerous things than waste processing systems. The Syndicate (and by extension, the Republic) is aware of this, and fines pilots involved.
+	illegal 50000
+	deadline 4
+	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
+	source "New Tortuga"
+	destination "La'ab Bamen"
+	on offer
+		event "know about Korath fish jobs"
+		conversation
+			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are turrets with heat signatures higher than a Breeder Reactor. Whoever built this place values their safety... and has the funds and technology to secure it.`
```

Comment:
Adaptive knowledge of heat.
```suggestion
			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are turrets with heat signatures higher than a Breeder Reactor. Whoever built this place values their safety... and has the funds and technology to secure it.`
				to display
					has "outfit (flagship installed): Thermal Scanner"
			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are huge turrets with heat emanating outwards. Whoever built this place values their safety... and has the funds and technology to secure it.`
				to display
					not "outfit (flagship installed): Thermal Scanner"
```

Reply:
No. This is too much duplicated text for such a minor thing. Also, the idea that a space ship who needs to know where to land wouldn't have infrared scanners is so unrealistic you're stretching suspension of disbelief to the breaking point.